yt/chyt/controller/internal/strawberry/controller.go

```diff
@@ -46,6 +46,13 @@ type Controller interface {
 	// Given speclet should have suitable type for the specific controller.
 	// Otherwise, GetOpBriefAttributes may panic.
 	GetOpBriefAttributes(parsedSpeclet any) map[string]any
+
+	// GetScalerTarget checks whether YT operation should be scaled and returns required scaling parameters.
+	// Return `nil` is scaling is not required.
+	GetScalerTarget(ctx context.Context, oplet *Oplet) (*ScalerTarget, error)
+
+	// NonRestartingOpSpecletFields returns "set" of fields that should not cause oplet restart when changed.
+	NonRestartingOpSpecletFields() map[string]struct{}
```

Comment:
I don't like this method in the interface.

I suggest to add a tag to the fields (like `yson:"abcd"`, but smth like `requires_restart:false`).
If there is such tag on the field, use it, otherwise assume that it `requires_restart`